templates/repo/view_file.tmpl

```diff
@@ -36,9 +36,11 @@
 		{{end}}
 	</h4>
 	<div class="ui attached table segment">
-		<div class="file-view {{if .IsRenderedHTML}}markdown{{else if .IsTextFile}}code-view{{end}} has-emoji">
-			{{if .IsRenderedHTML}}
+		<div class="file-view {{if .IsMarkup}}markdown{{else if .IsRenderedHTML}}plain-text{{else if .IsTextFile}}code-view{{end}} has-emoji">
+			{{if .IsMarkup}}
 				{{if .FileContent}}{{.FileContent | Str2html}}{{end}}
+			{{else if .IsRenderedHTML}}
+				<pre>{{if .FileContent}}{{.FileContent | Str2html}}{{end}}</pre>
```

Comment:
Move `<pre>` inside `{{if .FileContent}}` block. Don't need to render empty tag.

Reply:
@Morlinest it is better to render it because of margins as css is accounted for plain-text to have pre inside

Reply:
@lafriks OK then